src/app/package.json

```diff
@@ -73,6 +73,8 @@
     "react-redux": "~5.0.7",
     "react-router": "~4.2.0",
     "react-router-dom": "~4.2.2",
+    "react-scrollspy": "3.4.0",
+    "react-transition-group": "4.2.1",
```

Comment:
Let's remove `react-transition-group` since we didn't get around to it here. Reminder to rerun `./scripts/update` and chuck in the `yarn.lock` file